kbase-extension/static/kbase/js/common/cellComponents/filePathWidget.js

```diff
@@ -245,26 +314,18 @@ define([
                     title: span(['File Paths']),
                     name: `${cssClassType}s-area`,
                     body: [
-                        table(
-                            {
-                                class: `${cssBaseClass}__table`,
-                                dataElement: `${cssClassType}-fields`,
-                            },
-                            [
-                                tr({
```

Comment:
This was starting by adding an empty parameter row and doing tricky things to populate it later.